crypt.c

```diff
@@ -316,11 +322,15 @@ crypt_gensalt_rn (const char *prefix, unsigned long count,
 }
 SYMVER_crypt_gensalt_rn;
 
-#if INCLUDE_crypt_gensalt_r
 /* For code compatibility with older versions (v3.1.1 and earlier).  */
+#if INCLUDE_crypt_gensalt_r
 strong_alias (crypt_gensalt_rn, crypt_gensalt_r);
 SYMVER_crypt_gensalt_r;
 #endif
+#if INCLUDE_xcrypt_gensalt_r
```

Comment:
Make these `#if` blocks independent (so INCLUDE_xcrypt_gensalt_r=1 is effective even if INCLUDE_crypt_gensalt_r is zero).  Not because we'll ever do that; just because it's more consistent with how all the other compatibility aliases are written.

Reply:
Done in rebased commit.